src/components/lib/Footer.js

```diff
@@ -1,63 +1,90 @@
-      <TBTCLogo width="165" />
-    </div>
-    <nav className="footer-menu">
-      <div className="top-menu">
-        <ul>
-          <li>
-            <Link to="/about">
-              About
-            </Link>
-          </li>
-          <li>
-            <Link to="/news">
-              News
-            </Link>
-          </li>
-          <li>
-            <Link to="/developers">
-              Build
-            </Link>
-          </li>
-          <li>
-            <Link to="/faq">
-              FAQ
-            </Link>
-          </li>
-          <li>
-            <a href="https://dapp.test.tbtc.network/" target="_blank" rel="noopener noreferrer">
-              dAPP
-            </a>
-          </li>
-        </ul>
-      </div>
-      <div className="bottom-menu">
-        <ul>
-          <li>
-            <Link to="/privacy-policy">
-              Privacy Policy
-            </Link>
-          </li>
-          <li>
-            <Link to="/terms-of-use">
-              Terms of Use
-            </Link>
-          </li>
-          <li className="copyright">
-            &copy; 2020 tBTC
-          </li>
-        </ul>
+    <div className="container">
+      <div className="row justify-content-center no-gutters">
+        <div className="col-sm-12 col-lg-10">
+          <div className="footer-logo">
+            <TBTCLogo width="230" />
+          </div>
+          <nav className="footer-menu">
+            {navColumns.map((col, i) => (
+              <ul key={`nav-column-${i}`}>
+                {col.items.map((item, j) => (
+                  <li key={`nav-item-${j}`}>
+                    <Link to={item.url} className={item.icon ? "icon-link" : ""}>
+                      {item.icon ? <Image imageData={item.icon} /> : ""}
+                      {item.label}
+                    </Link>
+                  </li>
+                ))}
+              </ul>
+            ))}
+          </nav>
+        </div>
+        <div className="copyright col-sm-12 col-lg-10">
+          {copyright}
+        </div>
       </div>
-    </nav>
+    </div>
   </footer>
 )
 
+export const query = graphql`
+  query Footer {
+    allMarkdownRemark(
+      filter: { frontmatter: { template: { eq: "footer-nav" } } }
+    ) {
+      edges {
+        node {
+          fields {
+            locale
+          }
+          frontmatter {
+            nav_columns {
+              items {
+                label
+                url
+                icon {
+                  image {
+                    relativePath
+                  }
+                  alt
+                }
+              }
+            }
+            copyright_text
+          }
+        }
+      }
+    }
+  }
+`
+
+const Footer = ({ locale = "en" }) => (
+  <StaticQuery
+    query={query}
+    render={data => {
+      const match = data.allMarkdownRemark.edges
+        .find(e => e.node.fields.locale === locale)
+      return <FooterTemplate {...match.node.frontmatter} locale={locale} />
```

Comment:
I do wish we could represent `copyrightText` maybe as a parameter, and the columns as child elements. Obviously that would be a huge undertaking, so not pushing for it here :grin:

Reply:
That would be a subtle feat. :)